server/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportResizeAction.java

```diff
@@ -124,6 +125,16 @@ protected void masterOperation(
             return;
         }
 
+        // Index splits are not allowed for time-series indices
+        if (resizeRequest.getResizeType() == ResizeType.SPLIT) {
+            try {
+                IndexRouting.fromIndexMetadata(sourceMetadata).checkIndexSplitAllowed();
+            } catch (IllegalArgumentException e) {
+                listener.onFailure(e);
+                return;
+            }
+        }
```

Comment:
It doesn't look like the cleanest way to do it but it makes sense given what we have. And I don't think its permanent.